### PR TITLE
[FIX] don't break when an attribute node in an xpath makes the

### LINKED ISSUE #885: 8.0 Incompatibility between base_view_inheritance_extension and stock_packaging_usability_ul
If you try to install the module base_view_inheritance_extension with the module stock_packaging_usability_ul from here : https://github.com/OCA/stock-logistics-tracking
The installation fail.
I do not have the time to look at here more (maybe latter) , but as I have detected the issue I think it worth to share it.

### PR DESCRIPTION
**Pull request by hbrunn**
expression not match the node any more. Fixes #885 @sebastienbeau @StefanRijnhart

### CODE REVIEW

**Comment by hbrunn** [comment]
warning addressed in https://github.com/OCA/server-tools/pull/1020

File: base_view_inheritance_extension/models/ir_ui_view.py [hunk 1]
```diff
@@ -55,6 +55,9 @@ def _iter_inheritance_specs(self, spec):
                     yield node, handler
             return
         if spec.get('position') == 'attributes':
+            if all(not c.get('operation') for c in spec):
```
Comment (yajo) [inline base_view_inheritance_extension/models/ir_ui_view.py:58]: What if one has operation and another doesn't?

**Review by yajo — CHANGES_REQUESTED** [review]

File: base_view_inheritance_extension/models/ir_ui_view.py (hunk 1 above)
Comment (hbrunn) [reply base_view_inheritance_extension/models/ir_ui_view.py:58]: then it's a view which uses this module and then we can expect devs to order the child nodes in a way it doesn't break. Might be a good idea to talk about that in the readme.
The important issue this PR is meant to fix is that just installing this module breaks completely fine code like https://github.com/OCA/OCB/blob/10.0/addons/product/views/product_views.xml#L285

**Review by StefanRijnhart — APPROVED** [review]

**Review by sebastienbeau — APPROVED** [review]

**Comment by StefanRijnhart** [comment]
Oh, a bump of the version number is desired I believe?

**Comment by pedrobaeza** [comment]
Can you please change module version?

**Comment by hbrunn** [comment]
done

**Review by yajo — APPROVED** [review]

**Pull request merged**